tests/conftest.py

```diff
@@ -49,6 +47,7 @@ def _generate_app(
         # it is a installed app. Following monkeypatch make it more close
         # to the real scenario for test.
         monkeypatch.setattr(_AiidaLabApp, "is_installed", lambda _: True)
+        monkeypatch.setattr(_AiidaLabApp, "is_registered", lambda _: True)
```

Comment:
`is_registered` method fetches the app index, which we want to avoid in tests. This was likely the cause of the test flakiness that we saw in #413.